src/main/java/fr/insalyon/creatis/gasw/parser/GaswParser.java

```diff
@@ -308,8 +311,7 @@ public GaswInput getGaswInput(Map<String, String> inputsMap)
         return new GaswInput(executableName, parameters, downloads, uploads,
                 gaswVariables, envVariables);
     }
-
-    static String escapeSpecialBashCharacters(String stringToEscape) {
+        static String escapeSpecialBashCharacters(String stringToEscape) {
```

Comment:
put back the blank line and the good indentation

Reply:
done